web/src/site-admin/SiteAdminAddExternalServicesPage.tsx

```diff
@@ -17,45 +18,93 @@ interface Props extends ThemeProps {
-                <p className="mt-2">Add connections to non-code-host services.</p>
-                {Object.entries(nonCodeHostExternalServices).map(([id, externalService]) => (
-                    <div className="add-external-services-page__card" key={id}>
-                        <ExternalServiceCard to={getAddURL(id)} {...externalService} />
-                    </div>
-                ))}
+    return (
+        <div className="add-external-services-page mt-3">
+            <PageTitle title="Add repositories" />
+            <div className="d-flex justify-content-between align-items-center mt-3 mb-3">
+                <h2 className="mb-0">Add repositories</h2>
             </div>
-        )
-    }
+            <p className="mt-2">Add repositories from one of these code hosts.</p>
+            {!hasDismissedPrivacyWarning && (
+                <div className="alert alert-info">
+                    <p>
+                        This Sourcegraph installation will never send your code, repository names, file names, or any
+                        other specific code data to Sourcegraph.com or any other destination. Your code is kept private
+                        on this installation.
+                    </p>
+                    <h3>This Sourcegraph installation will access your code host by:</h3>
+                    <ul>
+                        <li>
+                            Periodically fetching a list of repositories to ensure new, removed, and renamed
+                            repositories are accessible on Sourcegraph.
+                        </li>
+                        <li>Cloning the repositories you specify to create a local cache.</li>
+                        <li>Periodically pulling cloned repositories to ensure search results are current.</li>
+                        <li>
+                            Fetching{' '}
+                            <a
+                                href="https://docs.sourcegraph.com/admin/repo/permissions"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                user repository access permissions
+                            </a>
+                            , if you have enabled this feature.
+                        </li>
+                        <li>
+                            Opening pull requests and syncing their metadata as part of{' '}
+                            <a
+                                href="https://docs.sourcegraph.com/user/campaigns"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                code change campaigns
+                            </a>
+                            , if you have enabled this feature.
+                        </li>
+                    </ul>
+                    <div className="d-flex flex-row-reverse">
```

Comment:
Reversing the order seems a bit of a round-about way to accomplish alignment to the end of the row :)

```suggestion
                    <div className="d-flex align-items-end">
```